### ISSUE TITLE
Unified build

### ISSUE THREAD

**Issue by Wechy77** [PAC-ROM/pac-rom#22]
I have problem with unified falcon and latest build commit. And I don't know how to solve it. This http://goo.gl/pGiQnB was added into build (cherry-picked from CM commits). I tried it before we merged it, and my builds always failed by cherry-pick it from cm gerrit. I figured it out, that the 4 lines after PAC text logo (http://goo.gl/vpBPZ6 ine 488-491 - "ro.product.device" and "ro.product.model" ) in ota_from_target_files is the problem. I tried many "if else combo" to keep it for not unified and to manage working for unified too, but i failed. Finally i deleted this 4 lines, and the build runs till end.

**Comment by Nick0703** [comment]
Please try again. The commit in build repo is old, I replaced it.

**Comment by Wechy77** [comment]
Now it's the same as in cm. It will also fail. 
My build.prop is this: http://pastebin.com/8EM6ZAJV
In my unified device build.prop  doesn't exists "ro.product.model", and "ro.product.device". It has "ro.build.product".
I don't know how to edit this 4 line http://pastebin.com/35WEsU79 to match the other changes in ota_from_target_files. If I leave it as it is, it says something that generated build.prop doesn't has <properties>. I tried to add "if OPTIONS.override_prop:" but I failed. Only solution (for me) was delete the 4 line. I think in our ota_from_target_files with this 4 lines we generate model and device in updater-script after PAC text logo.
